documentation/release_notes.rst

```diff
@@ -8,26 +8,23 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
-New in 2021.7.0
+New in 2021.7.1
 ===============
 
-Deprecation Notice
-------------------
-- Deprecated support of C++11 for Parallel API with host execution policies (``seq``, ``unseq``, ``par``, ``par_unseq``).
-  C++17 is the minimal required version going forward.
+New Features
+------------
+- Added namespace alias ``dpl = oneapi::dpl``.
+- Added a function that constructs a zip_iterator using a std::tuple of iterators.
```

Comment:
Maybe better "added possibility to construct a zip_iterator out of a std::tuple of iterators"?
@timmiesmith Is there a specification update patch for this?

Reply:
Applied suggestion